File: ios/RnOpenapp.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(RnOpenapp, NSObject)


RCT_EXTERN_METHOD(openApp:(NSString *)packageId
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

+ (BOOL)requiresMainQueueSetup
{
  return NO;
}

@end

File: ios/RnOpenapp.swift
@objc(RnOpenapp)
class RnOpenapp: NSObject {

    
    @objc(openApp:withResolver:withRejecter:)
    func openApp(packageId: String, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
        reject("Module not supported on this platform")
    }
}
